infrastructure/ansible/roles/dataset_loader/defaults/main.yml

```diff
@@ -108,6 +108,11 @@ dl_ds_default_users:
     email: userB+userA@kabletown.invalid
     fullName: A local account with RO rights
     role: read-only
+  - username: '{{ federation_user }}'
```

Comment:
This var needs a default value such as using something higher up like `dl_ds_default_federation_user`.  That way you don't have to apply defaults on every usage.

Reply:
Removed Federation User in f85db07bc7.

Reply:
Added default value `dl_ds_default_federation_user` in 7d64fdad49.